ddwrt_traffic.py

```diff
@@ -26,7 +27,7 @@ def parse_traffdata(content):
     result = []
     result_per_month = []
     for traffline in content:
-        if traffline.strip() == 'TRAFF-DATA':
+        if traffline.strip() == 'TRAFF-DATA' or traffline.strip() == "ÿ":
```

Comment:
Is this the only special character that can occur? Any idea where it comes from?

Reply:
> Is this the only special character that can occur? Any idea where it comes from?

This character is added at the last line of the file generated when i request a backup in the administration WebUI.

Reply:
And the character seams to be different on windows and linux client ...
So it is probably better to add `errors='ignore'` at line 8 in **ddwrt2csv.py**, like : 
`with open(file_source, errors='ignore') as f_src:`